libs/clients/official-journal-of-iceland/application/src/lib/ojoiApplicationClient.service.ts

```diff
@@ -122,23 +140,44 @@ export class OfficialJournalOfIcelandApplicationClientService {
   }
   async getPresignedUrl(
     params: GetPresignedUrlRequest,
+    auth: Auth,
   ): Promise<PresignedUrlResponse> {
-    return await this.ojoiApplicationApi.getPresignedUrl(params)
+    return await this.ojoiApplicationApiWithAuth(auth).getPresignedUrl(params)
   }
 
   async addApplicationAttachment(
     params: AddApplicationAttachmentRequest,
+    auth: Auth,
   ): Promise<void> {
-    await this.ojoiApplicationApi.addApplicationAttachment(params)
+    try {
+      console.log(auth)
+      await this.ojoiApplicationApiWithAuth(auth).addApplicationAttachment(
+        params,
+      )
+    } catch (error) {
+      this.logger.error('Failed to add application attachment', {
+        category: LOG_CATEGORY,
+        applicationId: params.id,
+      })
```

Comment:
Is this a true error? or just a warning? If isn't app breaking it should probably we .warn